spec/mailers/user_mailer_spec.rb

```diff
@@ -123,12 +123,6 @@
       expect(mail.html_part.body).to have_content(
         t('user_mailer.phone_changed.intro', app: APP_NAME)
       )
-
-      expect(mail.html_part.body).to have_content(
-        t('user_mailer.phone_changed.help',
-          app: APP_NAME,
-          contact_phone_number: CONTACT_PHONE_NUMBER)
-      )
```

Comment:
Yeah that's all I was thinking, in the past I know we had an issue with various links in the email not rendering properly (showing localhost for example), so just thought it would be go to double check the actual url was correct.